docs/track.rst

```diff
@@ -927,6 +924,122 @@ If ``detailed-results`` is ``true`` the following meta-data are returned in addi
+
+    .. note::
+        1. Parameters that are implicitly set by Rally (e.g. `body` or `request_cache`) are not supported (i.e. you should not try to set them and if so expect unspecified behavior).
+        2. Rally will not attempt to serialize the parameters and pass them as is. Always use "true" / "false" strings for boolean parameters (see example below).
+
+* ``body`` (mandatory): The query body.
+* ``pages`` (mandatory): Number of pages to retrieve (at most) for this search. If a query yields fewer results than the specified number of pages we will terminate earlier. To retrieve all result pages, use the value "all".
+* ``results-per-page`` (optional): Number of results to retrieve per page. This maps to the Search API's ``size`` parameter, and can be used for paginated and non-paginated searches. Defaults to ``10``
+* ``with-point-in-time-from`` (optional): The ``name`` of an ``open-point-in-time`` operation. Causes the search to use the generated `point in time <https://www.elastic.co/guide/en/elasticsearch/reference/current/point-in-time-api.html>`_.
+
+    .. note::
+        This parameter requires usage of a ``composite`` operation containing both the ``open-point-in-time`` task and this search.
+
+* ``response-compression-enabled`` (optional, defaults to ``true``): Allows to disable HTTP compression of responses. As these responses are sometimes large and decompression may be a bottleneck on the client, it is possible to turn off response compression.
+
+Example::
+
+    {
+      "name": "default",
+      "operation-type": "paginated-search",
+      "pages": 10,
+      "body": {
+        "query": {
+          "match_all": {}
+        }
+      },
+      "request-params": {
+        "_source_include": "some_field",
+        "analyze_wildcard": "false"
+      }
+    }
+
+.. note::
+    See also the ``close-point-in-time`` operation for a larger example.
+
+Throughput will be reported as number of retrieved pages per second (``pages/s``). The rationale is that each HTTP request corresponds to one operation and we need to issue one HTTP request per result page. Note that if you use a dedicated Elasticsearch metrics store, you can also use other request-level meta-data such as the number of hits for your own analyses.
+
+Meta-data
+"""""""""
+
+The following meta data are always returned:
+
+* ``weight``: "weight" of an operation. Always 1 for regular queries and the number of retrieved pages for scroll queries.
+* ``unit``: The unit in which to interpret ``weight``. Always "ops" for regular queries and "pages" for scroll queries.
+* ``success``: A boolean indicating whether the query has succeeded.
+* ``hits``: Total number of hits for this query.
+* ``hits_relation``: whether ``hits`` is accurate (``eq``) or a lower bound of the actual hit count (``gte``).
+* ``timed_out``: Whether the query has timed out. For scroll queries, this flag is ``true`` if the flag was ``true`` for any of the queries issued.
```

Comment:
This mentions scrolls when you want actually mean the point in time API? As we have a dedicated section for this API, I don't think we need to qualify this statement anyway. How about we simply say:

> ``timed_out``:  Whether any of the any of the queries issued has timed out.